2015/03-20/Chad/Challenge3.java

```diff
@@ -0,0 +1,73 @@
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics;
+import javax.swing.JFrame;
+import javax.swing.JComponent;
+
+public class FizzBuzz {
+	public static void FizzBuzzMethod(){
+		
+	}
```

Comment:
It's possible I forgot to remove a stub that I was playing with for images
instead of drawString.
On Apr 1, 2015 12:48 PM, "Pat Hawks" <notifications@github.com> wrote:

> In 2015/03-20/Chad/Challenge3.java
> <https://github.com/AICSC/Coding-Challenges/pull/15#discussion_r27604556>:
>
> > @@ -0,0 +1,73 @@
> > +import java.awt.Color;
> > +import java.awt.Font;
> > +import java.awt.Graphics;
> > +import javax.swing.JFrame;
> > +import javax.swing.JComponent;
> > +
> > +public class FizzBuzz {
> > +	public static void FizzBuzzMethod(){
> > +		
> > +	}
>
> This method looks to be never used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AICSC/Coding-Challenges/pull/15/files#r27604556>.
>